addons/website_form/static/src/snippets/s_website_form/001.js

```diff
@@ -250,7 +252,7 @@ publicWidget.registry.s_website_form_001 = publicWidget.Widget.extend({
 
     update_status: function (status) {
         if (status !== 'success') {  // Restore send button behavior if result is an error
-            this.$target.find('.s_website_form_send').removeClass('disabled');
+            self.$target.find('.s_website_form_send').removeClass('disabled');
```

Comment:
@sdegueldre Heuu ?

Reply:
Moved this line around and thought I didn't need it here, but I did so I put it back in and forgot I had to adapt it where I moved it. I'm surprised that didn't crash when I tested it.